### PR TITLE
Nevin/fix 1.0.0

### LINKED ISSUE #1616: 1.1 Missing string in the "Make default browser" notification
STR:
 1. Launch Rocket 
 2. Press settings 2ed time

Expected result:
 - Show "Make default browser" notification 

Actual result: 
 - Failed to show "Make default browser" notification . 
 - User have to swipe down the notification list to read the notification
 - Missing string is "Make Firefox Rocket as your default browser for fast browsing!"

Device: Redmi Note4
Rocket nightly: 1.1 (2862).debug.bitrise
Android version: 7.0
WebView version 64.0.3325.109

### LINKED ISSUE #1618: 1.1 Rocket should be able to open the shortcut immediately after adding a page to home screen
STR:
 1. Launch Rocket 
 2. Visite any webpage
 3. Tap menu
 4. Tap Add to Home screen button  
 5. Go to system Home 
 6. Tap the shortcut that you just added to home screen 

Expected result:
 - Rocket should be able to open the shortcut immediately 

Actual result: 
 - It will show the intent or open it via other app, e.g., Youtube 

Device: Redmi Note4
Rocket nightly: 1.1 (2862).debug.bitrise
Android version: 7.0
WebView version 64.0.3325.109

### LINKED ISSUE #1619: 1.1 Screenshot button disappeared
STR:
 1. Launch Rocket 
 2. Tap menu

Expected result:
 - There is a scissors button
 - Image: 
Image: screen shot 2018-03-20 at 6 20 33

Actual result: 
 - No scissors button
 - Image:  screenshot_2018-03-20-06-07-53-433_org mozilla rocket debug bitrise

Device: Redmi Note4
Rocket nightly: 1.1 (2862).debug.bitrise
Android version: 7.0
WebView version 64.0.3325.109r

### PR DESCRIPTION
**Pull request by cnevinc**
Fix #1616
Fix #1619
Fix #1618

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/notification/NotificationUtil.java [hunk 1]
```diff
@@ -30,7 +30,9 @@
                 .setColor(ContextCompat.getColor(context, R.color.surveyNotificationAccent))
                 .setAutoCancel(true)
                 .setContentIntent(pendingIntent)
-                .setPriority(NotificationCompat.PRIORITY_HIGH);
+                .setPriority(NotificationCompat.PRIORITY_HIGH)
+                // without setDefaults, the notification will not popup nor overlay
+                .setDefaults(NotificationCompat.DEFAULT_VIBRATE);
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/notification/NotificationUtil.java]: While I know we expect heads-up. Is it expected to vibrate?

File: app/src/main/java/org/mozilla/focus/utils/DialogUtils.java [hunk 2]
```diff
@@ -221,8 +221,8 @@ public static void showDefaultSettingNotification(Context context) {
         final PendingIntent openRocketPending = PendingIntent.getBroadcast(context, REQUEST_DEFAULT_CLICK, openDefaultBrowserSetting,
                 PendingIntent.FLAG_ONE_SHOT);
 
-        final String title = context.getString(R.string.app_name);
-        final String content = context.getString(R.string.preference_default_browser) + "?\uD83D\uDE4C";
+        final String title = context.getString(R.string.preference_default_browser) + "?\uD83D\uDE0A";
+        final String content = context.getString(R.string.notification_default_browser);
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/utils/DialogUtils.java]: This string is too long on my Pixel XL, we might need BigTextStyle?

File: app/src/main/res/layout/bottom_sheet_main_menu.xml [hunk 3]
```diff
@@ -145,11 +145,11 @@
         android:layout_height="1dp"
         android:background="@color/colorDivider" />
 
-    <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:layout_gravity="center_horizontal"
-        android:gravity="center_vertical">
+    <org.mozilla.focus.widget.EqualDistributeRow xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mTwTm) [inline app/src/main/res/layout/bottom_sheet_main_menu.xml]: Can't find the spec of this, Let's make sure with Carol.

Comment (cnevinc) [reply app/src/main/res/layout/bottom_sheet_main_menu.xml]: talked with Carol. I'll try to copy the spec from 2.0 for bottom menu
https://drive.google.com/file/d/1ccNdBlAYKekmlgg7f00cKHcJOb9GgRQl/view

File: app/src/main/java/org/mozilla/focus/utils/DialogUtils.java (hunk 2 above)
Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/utils/DialogUtils.java]: After discussion, we don't want this string for now (will be added in 2.0 with new string)

File: app/src/main/java/org/mozilla/focus/notification/NotificationUtil.java (hunk 1 above)
Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/notification/NotificationUtil.java]: Done

File: app/src/main/res/layout/bottom_sheet_main_menu.xml [hunk 4]
```diff
@@ -145,58 +145,72 @@
...
-                android:background="@drawable/action_capture"/>
-        </RelativeLayout>
-    </LinearLayout>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/fixed_menu_height"
+        android:gravity="center">
+
+        <org.mozilla.focus.widget.EqualDistributeRow
```
Comment (mTwTm) [inline app/src/main/res/layout/bottom_sheet_main_menu.xml]: I don't think we need this EqualDistributeRow?

**Pull request merged**